cosmosdb/sql/ps-account-firewall-create.ps1

```diff
@@ -1,6 +1,6 @@
 # Create an Azure Cosmos Account for SQL (Core) API with IP Firewall
```

Comment:
Unable to get this script to run. Errored out with:

```
New-AzResource : {"code":"BadRequest","message":"Unable to parse request payload. \r\nActivityId: f0cd3e0a-bda3-47de-acf3-e009c3f8ba8e, Microsoft.Azure.Documents.Common/2.5.1"}
At C:\Users\sttramer\src\azure-docs-powershell-samples\cosmosdb\sql\ps-account-firewall-create.ps1:26 char:1
+ New-AzResource -ResourceType "Microsoft.DocumentDb/databaseAccounts"  ...
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : CloseError: (:) [New-AzResource], ErrorResponseMessageException
    + FullyQualifiedErrorId : Microsoft.Azure.Commands.ResourceManager.Cmdlets.Implementation.NewAzureResourceCmdlet
```

Reply:
Found the issue with this error. Have corrected the script in the next commit.